src/colorizer/utils/url_utils.ts

```diff
@@ -53,40 +59,55 @@ export function fetchWithTimeout(
  * - `collection`: string path to the collection. Ignores paths matching the default collection address.
  * - `dataset`: string name or URL of the dataset.
  * - `feature`: string name of the feature.
- * - `track`: integer track number. Ignores values where `track < 0`.
- * - `time`: integer frame number. Ignores values where `time <= 0`.
+ * - `track`: integer track number.
+ * - `time`: integer frame number.
+ * - `thresholds`: array of feature threshold.
+ * - `range`: array of two numbers, representing the min and max of the color map range.
  *
  * @returns
  * - If no parameters are present or valid, returns an empty string.
  * - Else, returns a string of URL parameters that can be appended to the URL directly (ex: `?collection=<some_url>&time=23`).
  */
-export function stateToUrlParamString(state: Partial<UrlParams>): string {
+export function stateToUrlQueryString(state: Partial<UrlParams>): string {
   // arguments as more data gets stored to the URL.
 
   // Get parameters, ignoring null/empty values
   const includedParameters: string[] = [];
-  const { collection, dataset, feature, track, time } = state;
 
   // Don't include collection parameter in URL if it matches the default.
-  if (
-    collection &&
-    collection !== DEFAULT_COLLECTION_PATH &&
-    collection !== DEFAULT_COLLECTION_PATH + "/" + DEFAULT_COLLECTION_FILENAME
-  ) {
-    includedParameters.push(`${URL_PARAM_COLLECTION}=${encodeURIComponent(collection)}`);
+  if (state.collection) {
+    includedParameters.push(`${URL_PARAM_COLLECTION}=${encodeURIComponent(state.collection)}`);
   }
-  if (dataset) {
-    includedParameters.push(`${URL_PARAM_DATASET}=${encodeURIComponent(dataset)}`);
+  if (state.dataset) {
+    includedParameters.push(`${URL_PARAM_DATASET}=${encodeURIComponent(state.dataset)}`);
   }
-  if (feature) {
-    includedParameters.push(`${URL_PARAM_FEATURE}=${encodeURIComponent(feature)}`);
+  if (state.feature) {
+    includedParameters.push(`${URL_PARAM_FEATURE}=${encodeURIComponent(state.feature)}`);
   }
-  if (track && track >= 0) {
-    includedParameters.push(`${URL_PARAM_TRACK}=${track}`);
+  if (state.track !== undefined) {
```

Comment:
just curious, is this change because tracks can be negative?

Reply:
Tracks can't be negative... this change was more to remove some of the assumptions `url_utils` was making about the data format (making it a little simpler) and rely on the top-level App to handle that.